packages/mongodb.rb

```diff
@@ -29,7 +29,8 @@ class Mongodb < Package
   def self.build
     # prepare mongo-tools
     system "git clone https://github.com/mongodb/mongo-tools"
-    system "cd mongo-tools; ./build.sh ssl || echo '***YOU NEED TO SET UP GO ENVIRONMENT IN ORDER TO COMPILE MONGO-TOOLS***'"
+    system "cd mongo-tools; git checkout tags/r3.2.16; ./build.sh ssl || echo '***YOU NEED TO SET UP GO ENVIRONMENT IN ORDER TO COMPILE
+MONGO-TOOLS***'"
```

Comment:
Hey @jamesjnadeau: Thank you for the update.  It might be better if you used `&&` instead of `;` to separate the commands.  This will ensure that if the git checkout is not successful, it won't continue and try to execute build.sh.  Also, since this is a modification, please bump the version to something like `3.2.16-1` so users that have installed this version can get this update applied.  Otherwise, the change is not detected.
Thanks